packages/discovery-provider/integration_tests/tasks/entity_manager/test_playlist_entity_manager.py

```diff
@@ -824,20 +820,6 @@ def test_index_invalid_playlists(app, mocker):
                 )
             },
         ],
-        "UpdateAlbumTracklistUpdate": [
```

Comment:
this test case should no longer be valid - we allow users to update album tracklists now. not sure how this wasn't failing before.